riptide-stream/src/main/java/org/zalando/riptide/stream/StreamConverter.java

```diff
@@ -45,11 +45,10 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.type.TypeFactory;
 
-class StreamConverter<T> implements GenericHttpMessageConverter<T> {
+final class StreamConverter<T> implements GenericHttpMessageConverter<T> {
 
     private static final List<MediaType> DEFAULT_MEDIA_TYPES =
-            Arrays.asList(APPLICATION_JSON_SEQ,
-                    APPLICATION_X_JSON_STREAM);
+            Arrays.asList(APPLICATION_JSON_SEQ, APPLICATION_X_JSON_STREAM);
```

Comment:
This returns a fixed-size, **mutable** list which you're then returning in `getSupportedMediaTypes`, so someone could change it.